.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,54 @@
+name: ♿️ A11Y CI with Lighthouse
+on:
+  pull_request:
+    paths:
+      - '**/.github/workflows/lighthouse.yml'
+      - '**/.github/workflows/lighthouse-comment.js'
+      - '**/.github/workflows/lighthouse-budget.json'
+      - '.github/workflows/generate-site-preview.yml'
+      - 'src/**/*'
+      - 'package.json'
+      - 'package-lock.json'
+      - 'gatsby-*.js'
+      - '.nvmrc'
+
+env:
+  WEBSITE_TO_AUDIT: 'https://process-analytics-process-analytics-dev-site_preview-pr-${{ github.event.pull_request.number }}.surge.sh/'
+
+jobs:
+  Lighthouse-CLI:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      # Using the Lighthouse GitHub Action
+      # @see https://github.com/marketplace/actions/lighthouse-ci-action
+      - name: Run Lighthouse GitHub Action
+        uses: treosh/lighthouse-ci-action@v9
```

Comment:
❓ could we also check https://github.com/foo-software/lighthouse-check-action which seems to offer more options? in another workflow for example.